third_party/binutils/test_decenc/drdecode_decenc_x86_64.expect

```diff
@@ -102541,7 +102617,7 @@ test_x86_64_s:
  7b 00 00 00
  62 e2 7d 21 91 84 c5 vpgatherqd 0x0000007b(%rbp,%ymm16,8), %ymm16 {%k1} {%k1}
  7b 00 00 00
- 62 f1 7c 48 28 05 00 vmovaps <rel> 0x000000001006ffa6, %zmm0 {%k0}
+ 62 f1 7c 48 28 05 00 vmovaps <rel> 0x000000001007015a, %zmm0 {%k0}
```

Comment:
What caused this change?

Reply:
It's a pc-relative operand and the address of the instruction changed when I removed the #ifdef DISABLED_UNTIL_BUG_1312_IS_FIXED around avx512_vpopcntdq.s